modules/ddl-and-loading/pages/defining-a-graph-schema.adoc

```diff
@@ -256,40 +261,68 @@ CREATE DIRECTED EDGE::
 CREATE DIRECTED EDGE Edge_Type_Name "("
         FROM Vertex_Type_Name "," TO Vertex_Type_Name
    ["|" FROM Vertex_Type_Name, TO Vertex_Type_Name]*
+   [ ( attribute_name type DISCRIMINATOR ) |
+     ( DISCRIMINATOR "(" attribute_name ("," attribute_name)* ")" )]
    ["," attribute_name type [DEFAULT default_value]]* ")"
    [WITH REVERSE_EDGE="Rev_Name"]
 ----
 --
 ====
 
-A single edge type can be defined between multiple pairs of vertex types, e.g.
+A single edge type can be defined between multiple pairs of vertex types.
+For example:
 
-[source,gsql]
+[source.wrap,gsql]
 ----
-CREATE DIRECTED EDGE Member_Of (FROM Person, TO Org | FROM Org, TO Org,
-                               joined DATETIME)
+CREATE DIRECTED EDGE Member_Of (FROM Person, TO Org | FROM Org, TO Org, joined DATETIME)
 ----
 
-Note that edges do not have a `PRIMARY_ID` field.
-Instead, each edge is uniquely identified by a FROM vertex, a TO vertex, and optionally other attributes.
-The edge type may also be a distinguishing characteristic. For example, as shown in Figure 2 above, there are two types of edges between User and Book.  Therefore, both types would have attribute lists which begin `+(FROM User, To Book,...).+`
+=== `DISCRIMINATOR`
+The `DISCRIMINATOR` keyword defines a discriminator in an edge type definition to allow multiple instances of an edge type between two vertices.
+
+By default, edges are uniquely identified by the combination of the edge type, source vertex, and target vertex.
+Between two vertices, only one instance of a certain edge type is allowed to exist if the edge type definition does not include a discriminator.
+
+A discriminator is an attribute or a set of attributes that can be used to uniquely identify an edge.
+The discriminator must be unique between edges of the same type between two vertices.
+They *do not* have to be unique within the edge type as a whole.
+For example, an edge type `Study_At` with the `class_month` and `class_year` attributes as the discriminator can have multiple edge instances with the same `class_month` and `class_year`, as long as those edge instances are not between the same two vertices.
```

Comment:
```suggestion
For example, an edge type `Study_At` with the `class_month` and `class_year` attributes as discriminators can have multiple edge instances with the same `class_month` and `class_year`, as long as those edge instances are not between the same two vertices.
```

I think "study_at" as a discriminator example might not be as intuitive as a bank transfer example, because people don't tend to have multiple "study_at" data points but there could be many bank transfers.